assets/src/components/button/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Muriel-styled buttons.
+ */
+
+/**
+ * WordPress dependencies.
+ */
+import { Component } from '@wordpress/element';
+import { Button } from '@wordpress/components';
+
```

Comment:
The stylesheet needs to be imported here:

```suggestion
import './style.scss';
```